player/loadfile.c

```diff
@@ -259,31 +259,22 @@ static void print_stream(struct MPContext *mpctx, struct track *t)
     }
     char b[2048] = {0};
 
-    bool tracks_have_lang = false;
+    int max_lang_length = 0;
     for (int n = 0; n < mpctx->num_tracks; n++) {
-        if (mpctx->tracks[n]->lang) {
-            tracks_have_lang = true;
-            break;
-        }
+        if (mpctx->tracks[n]->lang)
+            max_lang_length = MPMAX(strlen(mpctx->tracks[n]->lang), max_lang_length);
     }
 
-    if (!isatty(STDOUT_FILENO)) {
-        APPEND(b, "%s ", t->selected ? BLACK_CIRCLE : WHITE_CIRCLE);
-    } else if (!t->selected) {
-        APPEND(b, "%s", TERM_ESC_GREY);
-    }
-    APPEND(b, "%-5s --%s=%-2d", tname, selopt, t->user_tid);
+    // Indent tracks after the Playing: line if present.
+    if (mpctx->playlist->num_entries > 1 || mpctx->playing->playlist_path)
```

Comment:
the list should be indented when printed for track switches, but with this condition it won't be
the calls are here:
```
player/command.c
1973:            print_track_list(mpctx, "Track switched:");
5928:                print_track_list(mpctx, "Track switched:");
5962:        print_track_list(mpctx, "Track added:");
5979:        print_track_list(mpctx, "Track removed:");
6017:    print_track_list(mpctx, "Reloaded:");
6040:        print_track_list(mpctx, "Track list:");
7330:    print_track_list(mpctx, "Track switched:");
```